.travis.yml

```diff
@@ -1,14 +1,23 @@
+sudo: required
+dist: trusty
+addons:
+  apt:
+    sources:
+      - google-chrome
+    packages:
+      - google-chrome-stable
 language: node_js
 node_js:
-  - "node"
```

Comment:
Not particular reason. We already testing for node 4 and 6.
`node` is the same as `6`.
That's why I removed that.